EventListener/LogoutListener.php

```diff
@@ -35,9 +35,9 @@ public function __construct(
         $this->defaultTargetRouteName = $defaultTargetRouteName;
     }
 
-    public function onSymfonyComponentSecurityHttpEventLogoutEvent(LogoutEvent $event)
+    public function onSymfonyComponentSecurityHttpEventLogoutEvent(LogoutEvent $event): void
     {
-        if (null === $event->getToken() || null === $event->getToken()->getUser()) {
+        if (null === $event->getToken()?->getUser()) {
```

Comment:
Not a good idea to use the null safe operator (uncompatible with < php8)

Reply:
hmmm Symfony 6.4 requires >= PHP 8.1